### DISCUSSION TITLE
Peer authentication?

### DISCUSSION THREAD

**Discussion by hongquan** [geldata/gel#6534] [Ideas]
PostgreSQL has [peer authentication](https://www.postgresql.org/docs/current/auth-peer.html), which is useful for single-server deployment. In EdgeDB documentation, I don't find anywhere mentioning it. Does it exist in EdgeDB?

**Comment by elprans** [comment]
Peer auth only works with Unix socket connections, EdgeDB currently only supports TCP connections (outside of the special admin connection socket used by the CLI).
